src/04_modality_specific_files/01_magnetic_resonance_imaging_data.md

```diff
@@ -1,97 +1,9 @@
-Magnetic Resonance Imaging data
--------------------------------
+# Magnetic Resonance Imaging data
```

Comment:
this also applies to the MEG section below. Since this is listed as a subheading (h2) under the Modality Specific Files heading (existed only at the folder level right now), should this be marked as an h1 here?

Reply:
That throws off mkdocs (might be the same effect on mdbook):

with `## Behavioral experiments (with no MRI)`
![image](https://user-images.githubusercontent.com/238759/46453253-624a8a80-c755-11e8-859f-99f99ba1bbb8.png)

with `# Behavioral experiments (with no MRI)`
![image](https://user-images.githubusercontent.com/238759/46453265-75f5f100-c755-11e8-9dec-05e66d753a65.png)